### PR TITLE
Revert 1 ap isheet

### PR DESCRIPTION
**Pull request by aedward8**
I accidentally merged the PR before I was ready, so now I am comparing the "reverted" branch (API revert, one commit before the merged occurred? I believe this is what is happening as I am comparing to API sheet)

In this PR I successfully moved the fetch API call from the React side to Express side , so the fetch url is no longer displayed to the user !! woot woot. Please help me review and see if there are some obvious code re-factoring/ cleaner way of writing things.

Thank you

### CODE REVIEW

**Review by alexcheva — APPROVED** [review]
Running into an issue with `npm run db:init` while trying to run the branch locally, so the add task button does not work. But it looks good, I have issues with Docker on my end.

File: app/src/App.js [hunk 1]
```diff
@@ -54,89 +53,4 @@ const AddTask = ({ loadTasks }) => {
   );
 };
 
-const Form = () => {
```
[the remark this answers is not in the document]
Comment (aedward8) [reply app/src/App.js:57]: yes I moved all components to separate files now :)

File: server/package.json [hunk 2]
```diff
@@ -11,11 +11,9 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "body-parser": "^1.19.0",
```
[the remark this answers is not in the document]
Comment (aedward8) [reply server/package.json:14]: I realized express has it's own body parser so I uninstalled the extra package

**Comment by aedward8** [comment]
PR is completed from review

**Pull request closed without merging**